pkg/monitor/api/types.go

```diff
@@ -434,3 +434,45 @@ func ServiceDeleteRepr(
 	repr, err := json.Marshal(notification)
 	return string(repr), err
 }
+
+const (
+	// PolicyIngress is the value of Flags&PolicyNotifyFlagDirection for ingress traffic
+	PolicyIngress = 1
+
+	// PolicyEgress is the value of Flags&PolicyNotifyFlagDirection for egress traffic
+	PolicyEgress = 2
+
+	// PolicyMatchNone is the value of MatchType indicatating no policy match
+	PolicyMatchNone = 0
```

Comment:
is this meant to be of type `PolicyMatchType` and have iota?

Reply:
Our policy is not to fix code while moving it to assist in reviews.